input/drivers/udev_input.c

```diff
@@ -4215,13 +4360,62 @@ static void udev_input_grab_mouse(void *data, bool state)
 #endif
 }
 
+
+static bool udev_input_set_sensor_state (void *data, unsigned port, enum retro_sensor_action action, unsigned rate) {
+   /*TODO*/
+   RARCH_DBG("udev_input_set_sensor_state: %d %d %d\n", port, action,rate);
+   return true;
+}
```

Comment:
This looks like it's leftover from before my luminance sensor support was merged?